test/dead_link_test.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:link_checker/link_checker.dart';
+import 'package:test/test.dart';
+
+void main() => test('project for dead links', () async {
+      var badLinks = <BadLink>[];
+      await for (BadLink badLink
+          in getBadLinksInDirectory(blacklistedLinksRegexes: [
```

Comment:
`getBadLinksInDirectory` assumes that `Directory.current` is the project root. I don't know if this always holds? It might...